client/payment-details/timeline/map-events.js

```diff
@@ -415,6 +415,18 @@ export const feeBreakdown = ( event ) => {
 						'Subscription transaction fee: %1$s%%',
 						'woocommerce-payments'
 				  ),
+		'additional-device':
+			0 !== fixedRate
```

Comment:
Nice work adding both fixed and % rates even if we only have one for now! Makes future updates unnecessary 🙇